app/requirements.txt

```diff
@@ -1,3 +1,5 @@
 pydantic==1.9.1

```

Comment:
looks like the requirements for "jose" is missing.

Reply:
added in requirements.txt